src/main/java/gregtech/api/recipes/RecipeMaps.java

```diff
@@ -843,4 +843,28 @@
     @ZenProperty
     public static final FuelRecipeMap PLASMA_GENERATOR_FUELS = new FuelRecipeMap("plasma_generator")
             .setSound(GTSounds.TURBINE);
+
+    @ZenProperty
+    public static final RecipeMap<FuelRecipeBuilder> COMBUSTION_GENERATOR_FUELS_2 = new RecipeMap<>("combustion_generator_2", 0, 0, 0, 0, 1, 1, 0, 0, new FuelRecipeBuilder(), false)
```

Comment:
Why the renames of these RecipeMaps with `_2`? Couldn't they replace the old RecipeMaps, or get a better name

Reply:
Too many other things use the old system, and entirely depend on it at present. That is why this is only a part 1. Other things, such as large turbines, armor changes and the large boilers need to be finished before the original FuelRecipeMaps can be removed.